### PR TITLE
FUSETOOLS-1463 - brute force workaround for test failure

### PR DESCRIPTION
**Pull request by bfitzpat**

### CODE REVIEW

**Comment by jpav** [comment]
Is there a reason to restore this property back to its original value?  I know it’s a new level of “protection”, but we’ve never had this protection before, and anyone that needs to avoid the error will just set the property anyways.  Seems like this would be better done in the constructor without worrying about the restore.

**Comment by bfitzpat** [comment]
I'm good with setting it in the POM, but couldn't figure out a way to do it at our level. As for setting the value back, I like cleaning up my mess.

**Comment by jpav** [comment]
I certainly understand the idea of restoring things back to the way they were before messing with them, but to me this is a mess that Java put in place to begin with.  If we set it in the POM, it would more appropriately be done in the parent POM above us, which is what I had in mind.  If you do keep the current approach of doing it in the code, then we should at least set and unset the property within each method (I thought I remember the PR setting it in generateFromInstance and unsetting it in generateFromSchema).

**Comment by bfitzpat** [comment]
Yeah, I don't think my fix is the best approach by any stretch. If we can do it in the pom, that would be better.

File: transformation/common/src/main/java/org/jboss/tools/fuse/transformation/model/xml/XmlModelGenerator.java [hunk 1]
```diff
@@ -214,6 +216,12 @@ public JCodeModel generateFromInstance(File instanceFile,
                                            File schemaFile,
                                            String packageName,
                                            File targetPath) throws Exception {
+    	// Step 0 - avoid issue with schema loading on Java 8
+    	if (System.getProperty("java.xml.accessExternalSchema") != null) {
```
Comment (apupier) [inline transformation/common/src/main/java/org/jboss/tools/fuse/transformation/model/xml/XmlModelGenerator.java]: is it normal to use java without x here?

File: transformation/common/src/main/java/org/jboss/tools/fuse/transformation/model/xml/XmlModelGenerator.java [hunk 2]
```diff
@@ -214,6 +216,12 @@ public JCodeModel generateFromInstance(File instanceFile,
...
+    		_accessExternalSchemaValue = System.getProperty("java.xml.accessExternalSchema");
+    	}
+        System.setProperty("javax.xml.accessExternalSchema", "all");
```
Comment (apupier) [inline transformation/common/src/main/java/org/jboss/tools/fuse/transformation/model/xml/XmlModelGenerator.java]: Is it this property? https://docs.oracle.com/javase/7/docs/api/javax/xml/XMLConstants.html#ACCESS_EXTERNAL_SCHEMA
Shouldn't we use the constant?

Comment (apupier) [inline transformation/common/src/main/java/org/jboss/tools/fuse/transformation/model/xml/XmlModelGenerator.java]: the constant seems not working

**Comment by apupier** [comment]
I included the modification in https://github.com/fusesource/fuseide/pull/465 because I needed it for fixing the test

**Pull request closed without merging**